libraries/aws_elb.rb

```diff
@@ -14,53 +14,71 @@ class AwsElb < AwsResourceBase
     super(opts)
     validate_parameters(required: [:load_balancer_name])
-
-    catch_aws_errors do
-      load_balancer_name = { load_balancer_names: [opts[:load_balancer_name]] }
-      resp = @aws.elb_client.describe_load_balancers(load_balancer_name).load_balancer_descriptions.first
-      @attrs = @aws.elb_client.describe_load_balancer_attributes(load_balancer_name: opts[:load_balancer_name]).load_balancer_attributes
-
-      @availability_zones = resp.availability_zones
-      @dns_name           = resp.dns_name
-      @load_balancer_name = resp.load_balancer_name
-      @listeners          = resp.listener_descriptions
-      @external_ports     = resp.listener_descriptions.map { |l| l.listener.load_balancer_port }
-      @instance_ids       = resp.instances.map(&:instance_id)
-      @internal_ports     = resp.listener_descriptions.map { |l| l.listener.instance_port }
-      @security_group_ids = resp.security_groups
-      @subnet_ids         = resp.subnets
-      @vpc_id             = resp.vpc_id
-      @protocols          = resp.listener_descriptions.map { |l| l.listener.protocol }.uniq
-      @certificate_id     = resp.listener_descriptions.map { |l| l.listener.ssl_certificate_id }
-
-      # The ELB will list multiple custom policies, including previously configured policies
-      # Even if a pre-defined policy is selected, a custom policy is created from that template
-      # So, we need to get a list of all policies currently in use and filter the list of policies that exist
-      policies_in_use     = resp.listener_descriptions.map(&:policy_names).flatten.uniq
-      @ssl_policies       = @aws.elb_client.describe_load_balancer_policies(load_balancer_name: opts[:load_balancer_name])
-                                .policy_descriptions.select { |p| policies_in_use.include?(p.policy_name) }
-    end
+    @availability_zones = load_balancer_description.availability_zones
+    @dns_name           = load_balancer_description.dns_name
+    @load_balancer_name = load_balancer_description.load_balancer_name
+    @external_ports     = load_balancer_description.listener_descriptions.map { |l| l.listener.load_balancer_port }
+    @instance_ids       = load_balancer_description.instances.map(&:instance_id)
+    @internal_ports     = load_balancer_description.listener_descriptions.map { |l| l.listener.instance_port }
+    @security_group_ids = load_balancer_description.security_groups
+    @subnet_ids         = load_balancer_description.subnets
+    @vpc_id             = load_balancer_description.vpc_id
+    @listeners          = load_balancer_description.listener_descriptions
+    @protocols          = load_balancer_description.listener_descriptions.map { |l| l.listener.protocol }.uniq
+    @certificate_id     = load_balancer_description.listener_descriptions.map { |l| l.listener.ssl_certificate_id }
   end
 
   def exists?
-    !@vpc_id.nil?
+    !vpc_id.nil?
   end
 
   def to_s
     "AWS ELB #{load_balancer_name}"
```

Comment:
```suggestion
    "ELB Name: #{load_balancer_name}"
```

Reply:
so this was the previous convention and I don't want to change this!